Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml

```
@@ -131,6 +131,8 @@
       gender: male
   - type: Sharp
   - type: PotentialPsionic # Nyano
+  - type: NightVision #Delta-v - Added nightvision to rat king
+    drawOverlay: false
```

Suggestion: Make the color not green. The color in the suggestion is a grey color.

```suggestion
  - type: NightVision #Delta-v - Added nightvision
    drawOverlay: false
    color: "#808080"
```

---

"drawOverlay: false" disables the color overlay and just makes it bright like in ghost vision. Though i can check if it looks better with a differently colored overlay

---

So its full color?